src/TestExplorer/TestExplorer.ts

```diff
@@ -84,9 +84,13 @@ export class TestExplorer {
      */
     async discoverTestsInWorkspace() {
         try {
-            const { stdout } = await execSwift(["test", "--skip-build", "--list-tests"], {
-                cwd: this.folderContext.folder.fsPath,
-            });
+            const enableTestDiscovery = await testDiscoveryFlag(this.folderContext);
+            const { stdout } = await execSwift(
```

Comment:
You shouldn't need the test discovery flag here.

Reply:
🤔Oh, I didn’t see that it is automatically triggered after build tasks.

Yet I’m still wondering whether it makes sense to append `--skip-build` here. Does it mean that a user needs to trigger a build task manually so as to see the tests?

Reply:
If you have a large project, which needs completely rebuilt to get a list of tests, all the user will see is an empty test view for ages. I was going to try and add a message to say "please build your project if you want to view tests".

Reply:
I see. Gonna resolve this after my sleep:)